app/controllers/wikis_controller.rb

```diff
@@ -46,9 +46,14 @@ def new
   end
 
   def edit
+    @user = current_user
+    @wiki = Wiki.find(params[:id])
+    @user_emails = User.where.not(id: current_user.id || @wiki.users.pluck(:id)).map(&:email)
```

Comment:
@wiki.users.pluck(:id) will return an array of integers (user ids) but then you are calling .email on every instance in that collection which should raise an error since there is no method called `email` in the integer class.